content/blog/censored-0-2-0/index.Rmd

```diff
@@ -0,0 +1,158 @@
+* [x] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [x] `usethis::use_tidy_thanks()`
+-->
+
+We're thrilled to announce the release of [censored](https://censored.tidymodels.org/) 0.2.0. censored is a parsnip extension package for survival models.
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("censored")
+```
+
+This blog post will introduce you to a new argument name, `eval_time`, and two new engines for fitting random forests and parametric survival models.
+
+You can see a full list of changes in the [release notes](https://github.com/tidymodels/censored/releases/tag/v0.2.0).
+
+```{r setup, include = FALSE}
+library(tidymodels)
+library(censored)
+tidymodels_prefer()
+theme_set(theme_minimal())
+```
+
+## Introducting `eval_time`
+
+As we continue to add support for survival analysis across tidymodels, we have seen a need to be more explicit about which time we mean when we say "time": event time, observed time, censoring time, time at which to predict survival probability at? The last one is a particular mouthful. We now refer to this time as "evaluation time". In preparation for dynamic survival performance metrics which can be calculated at different evaluation time points, the argument to set these evaluation time points for `predict()` is now called `eval_time` instead of just `time`.
+
+```{r}
+cox <- proportional_hazards() |> 
+  set_engine("survival") |> 
+  set_mode("censored regression") |> 
+  fit(Surv(time, status) ~ ., data = lung)
+pred <- predict(cox, lung[1:3, ], type = "survival", eval_time = c(100, 500))
+pred
+```
+
+The predictions follow the tidymodels principle of one row per observation, and the nested tibble contains the predicted survival probability, `.pred_survival`, as well as the corresponding evaluation time. The column for the evaluation time is now called `.eval_time` instead of `.time`.
+
+```{r}
+pred$.pred[[2]]
+```
+
+## New engines
+
+censored contains engines for parametric, semi-parametric, and tree-based models. This release adds two new engines:
+
+- the `"aorsf"` engine for random forests via `rand_forest()`
+- the `"flexsurvspline"` engine for parametric models via `survival_reg()`
+
+### New `"aorsf"` engine for `rand_forest()`
+
+This engine has been contributed by [Byron Jaeger](https://github.com/bcjaeger) and enables users to fit oblique random survival forests with the aorsf package. What's with the oblique you ask?
+
+Oblique describes how the decision trees that form the random forest make their splits at each node: If the split is based on a single predictor, the resulting tree is called _axis-based_ because the split is perpendicular to the axis of the predictor. If the split is based on a linear combination of predictors, there is a lot more flexibility in how the data is split: the split does not need to be perpendicular to any of the predictor axes. Such trees are called _oblique_. 
```

Comment:
```suggestion
Oblique describes how the decision trees that form the random forest make their splits at each node. If the split is based on a single predictor, the resulting tree is called _axis-based_ because the split is perpendicular to the axis of the predictor. If the split is based on a linear combination of predictors, there is a lot more flexibility in how the data is split: the split does not need to be perpendicular to any of the predictor axes. Such trees are called _oblique_. 
```

Reply:
or lowercase the "If" following the colon